events/syscall/detect_outbound_connections_to_common_miner_pool_ports.go

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"os/exec"
+
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+	DetectOutboundConnectionsToCommonMinerPoolPorts,
+	events.WithDisabled(), // this rules is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func DetectOutboundConnectionsToCommonMinerPoolPorts(h events.Helper) error {
+	curl, err := exec.LookPath("curl")
+	if err != nil {
+		h.Log().Warnf("Curl is needed to launch this action")
+		return err
+	}
+	cmd := exec.Command(curl, "http://mine.moneropool.com")
```

Comment:
Mmh is GETting this link safe?

Reply:
Dont have an idea about that 🥲🥲. I executed in my system 🙃

Reply:
cc @leogr

Reply:
@FedeDP @leogr if we are worrying about this 
> Mmh is GETting this link safe?

then why cant we follow this approach i.e, let us add any example domain in the rule [here](https://github.com/falcosecurity/rules/blob/88a40c8d9c9af7985998689e9847f60a2a5de898/rules/falco-sandbox_rules.yaml#L1439-L1449), which can be used for testing purposes.  So after that we can curl example.com which triggers the rule and safe to GET the request. WDYT? about this approach. IMO instead of curling the suspicious  domains we can do this which will be beneficial for testing purposes

```
- list: http_miner_domains
  items: [
    "ca.minexmr.com",
    "de.minexmr.com",
    "fr.minexmr.com",
    "mine.moneropool.com",
    "mine.xmrpool.net",
    "pool.minexmr.com",
    "sg.minexmr.com",
    "xmr.crypto-pool.fr",
    "example.com" # Not a miner domain but used for testing purposes in falco
  ]

```

Reply:
I'm sorry. I had lost track of this. Anyway, adding `example.com` to the list does not seem a viable option to me since it may increase false positives.